### DISCUSSION TITLE
Unable to see guardrails trace on langfuse

### DISCUSSION THREAD

**Discussion by Ishika2005** [NVIDIA-NeMo/Guardrails#601] [Q&A]
I am currently unable to view guardrails traces on Langfuse. Below is the relevant part of my code where I create the RAG chain:

```python
rag_chain_with_guardrails = (
    RunnableParallel(context=retriever | format_docs, question=RunnablePassthrough())
    | review_template
    | (guardrails | llm)
    | StrOutputParser()
)
```
And here is the code to call rag_chain_with_guardrails with input:

```python
response=rag_chain_with_guardrails.invoke(question,config={"callbacks": [langfuse_handler]})
```
Below is a screenshot showing the relevant configuration:

[
<img width="817" alt="image" src="https://github.com/NVIDIA/NeMo-Guardrails/assets/84271695/ff554f9e-396b-4e54-8489-978b6cda961d">
](url)

Could you please assist me in resolving this issue? If any additional information is needed, please let me know.

**Comment by dan-robinson-ai** [comment]
same issue
